### ISSUE TITLE
Switchery for dynamically inserted elements

### ISSUE THREAD

**Issue by Snoopy68** [abpetkov/switchery#43]
Hello.

I add some new input/checkbox elements to my website dynamically by jquery/before().
Those inputs are already looking like a switchery element ... don't know why, but ok.

BUT:

They are not reaction on user-clicks. Absolutly nothing.
The input elements have onchange-events like every other (working) switchery-input on the site.

**Comment by ChickenSniper** [comment]
I'm having the same issues...I'm assuming this isn't supported?

**Comment by justein** [comment]
you can acquire the check state by binding Listener on it

**Comment by abpetkov** [comment]
Honestly, from what you've explained it looks more like a problem of your implementation. An live example might help here. If you're through this or resolved this differently, please share if you did and/or how, so we can try to solve this issue.

**Comment by minieggs** [comment]
I think I have a similar issue.

I have downloaded your switchery.css/js and have them working fine in a standalone webpage

I'm no JS expert and I'm now trying to use them in a proprietary framework

I notice that the js file is included at the end of the html file.

I assume this is because the checkbox elements need to exist before the script executes ?

The framework I'm trying to use it in creates everything dynamically and i'm getting an error on line 1905 of switchery.js


`Switchery.prototype.isDisabled = function() {
  return this.options.disabled || this.element.disabled || this.element.readOnly;
};`

Basically it is saying element is null and therefore it can't get the disabled property

Can you give me some simple clues on how i might use your switchery styling with my dynamically created checkboxes

Thanks
